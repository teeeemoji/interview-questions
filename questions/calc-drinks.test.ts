import calcDrinks from './calc-drinks'

test('3', () => {
    expect(calcDrinks(3)).toMatchSnapshot()
})

test('10', () => {
    expect(calcDrinks(10)).toMatchSnapshot()
})