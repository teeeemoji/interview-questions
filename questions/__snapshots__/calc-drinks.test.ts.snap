// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`3 1`] = `1`;

exports[`10 1`] = `5`;
